pom.xml

```diff
@@ -286,6 +286,22 @@
       <artifactId>wiremock-standalone</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+      <version>2.3.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-core</artifactId>
+      <version>2.3.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-impl</artifactId>
+      <version>2.3.0</version>
+    </dependency>
```

Comment:
These are all to fix missing class complaints, I would have used version 2.2.11 but this gives the exception 
```
[ERROR] Failed to execute goal org.jenkins-ci.tools:maven-hpi-plugin:2.1:test-hpl (default-test-hpl) on project nexus-jenkins-plugin: Error preparing the manifest: Failed to open artifact com.sun.xml.bind:jaxb-core:jar:2.2.11:compile at C:\Users\Richy\.m2\reposit
ory\com\sun\xml\bind\jaxb-core\2.2.11\jaxb-core-2.2.11.jar: Some problems were encountered while processing the POMs:
[ERROR] [ERROR] 'dependencyManagement.dependencies.dependency.systemPath' for com.sun:tools:jar must specify an absolute path but is ${tools.jar} @ com.sun.xml.bind.mvn:jaxb-parent:[unknown-version], C:\Users\Richy\.m2\repository\com\sun\xml\bind\mvn\jaxb-parent\
2.2.11\jaxb-parent-2.2.11.pom, line 188, column 29
```